doc/src/siteconfig.json

```diff
@@ -15,69 +15,7 @@
     "terms": {
       "label": "Terms of Service",
       "link": "https://sui.io/terms/"
-    },
-    "menu": [
-      {
-        "title": "Docs",
-        "listItems": [
-          {
-            "title": "Getting Started",
-            "link": "/build/move"
-          },
-          {
-            "title": "API Reference",
-            "link": "https://docs.sui.io/sui-jsonrpc",
-            "external": true
-          },
-          {
-            "title": "Examples",
-            "link": "https://examples.sui.io/",
-            "external": true
-          }
-        ]
-      },
-      {
-        "title": "Community",
-        "listItems": [
-          {
-            "title": "Discord",
-            "link": "https://discord.gg/sui",
-            "external": true
-          },
-          {
-            "title": "Twitter",
-            "link": "https://twitter.com/SuiNetwork/",
-            "external": true
-          },
-          {
-            "title": "Medium",
-            "link": "https://medium.com/mysten-labs/",
-            "external": true
-          }
-        ]
-      },
-      {
-        "title": "Team",
-        "listItems": [
-          {
-            "title": "GitHub",
-            "link": "https://github.com/MystenLabs",
-            "external": true
-          },
-          {
-            "title": "Sui.io",
-            "link": "https://sui.io/",
-            "external": true,
-            "arrow": true
-          },
-          {
-            "title": "White Paper",
-            "link": "https://github.com/MystenLabs/sui/blob/main/doc/paper/sui.pdf",
-            "external": true
-          }
-        ]
-      }
-    ]
+    }
```

Comment:
This is taking out the footer menu 
<img width="1998" alt="Screen Shot 2022-10-19 at 7 27 08 PM" src="https://user-images.githubusercontent.com/8676844/196823116-6c4559d1-c218-4c4b-aaf2-ace02be27350.png">

Reply:
Correct